src/CoreWCF.WebHttp/src/CoreWCF/Description/DataContractJsonSerializerOperationBehavior.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Json;
+using System.Xml;
+
+namespace CoreWCF.Description
+{
+    // TODO: Is this right.
+
+    //internal class DataContractJsonSerializerOperationBehavior : DataContractSerializerOperationBehavior
+    //{
+    //    private readonly bool _alwaysEmitTypeInformation;
+
+    //    public DataContractJsonSerializerOperationBehavior(OperationDescription description, int maxItemsInObjectGraph, bool ignoreExtensionDataObject, IDataContractSurrogate dataContractSurrogate, bool alwaysEmitTypeInformation)
+    //        : base(description)
```

Comment:
There's a new surrogate interface type, but I'm hesitant to add code for using it as we might need to bring the original one back because it has support for exporting the schema (needed for wsdl) that the new interface doesn't. Until that's resolved, simply ignoring the surrogate provider for now if probably best.

Reply:
This has been addressed.